client/src/modules/display_metadata/display_metadata.html

```diff
@@ -0,0 +1,121 @@
+                      DisplayMetadataCtrl.filterElements,
+                      DisplayMetadataCtrl.patient.uuid,
+                      DisplayMetadataCtrl.patient) }}">
+                    <span class="fa fa-file-pdf-o"></span> <span translate>DOWNLOADS.PDF</span>
+                  </a>
+                </li>
+                <li role="separator" class="divider"></li>    
+                <!-- FIX ME TO DOWMLOAD AS CSV and EXCEL 
+                <li role="menuitem">
+                  <a>
+                    <span class="fa fa-file-excel-o"></span> <span translate>DOWNLOADS.CSV</span>
+                  </a>
+                </li>
+    
+                <li role="menuitem">
+                  <a>
+                    <span class="fa fa-file-excel-o"></span> <span translate>DOWNLOADS.EXCEL</span>
+                  </a>
+                </li> -->
+    
+              </ul>
+            </div>
+          </div>
+
+      <div ng-if="DisplayMetadataCtrl.patientData" class="toolbar-item">
+        <button class="btn btn-default text-capitalize" id="create" ui-sref="display_metadata.patientfill({id : DisplayMetadataCtrl.collector.id, patient: DisplayMetadataCtrl.patient.uuid})" data-method="create">
+          <span class="fa fa fa-file-text"></span> <span class="hidden-xs" translate>FORM.BUTTONS.FILL_FORM</span>
+        </button>
+      </div>
+
+      <div ng-if="DisplayMetadataCtrl.patientData" class="toolbar-item">
+        <button class="btn btn-default text-capitalize" id="create" ui-sref="patientRecord({ patientUuid : DisplayMetadataCtrl.patient.uuid })" data-method="create">
+          <span class="fa fa-medkit"></span> <span class="hidden-xs" translate>FORM.LABELS.MEDICAL_SHEETS</span>
+        </button>
+      </div>
+
+      <div class="toolbar-item">
+        <button
+          ng-click="DisplayMetadataCtrl.search()"
+          data-method="search"
+          class="btn btn-default">
+          <span class="fa fa-search"></span> <span class="hidden-xs" translate>FORM.BUTTONS.SEARCH</span>
+        </button>
+      </div>
+
+      <div class="toolbar-item">
+        <button
+          type="button"
+          ng-click="DisplayMetadataCtrl.toggleFilter()"
+          data-method="filter"
+          class="btn btn-default"
+          ng-class="{ 'btn-info' : DisplayMetadataCtrl.filterEnabled }">
+          <i class="fa fa-filter"></i>
+        </button>
+      </div>
+    </div>
+  </div>
+</div>
+
+<!-- main content -->
+<div class="flex-util" style="overflow: auto" ng-if="DisplayMetadataCtrl.collector">
+  <span data="filter">
+    <span class="label" style="text-shadow: {{ DisplayMetadataCtrl.collector.color }} 5px 3px 8px; font-size: 15px; color: black">
+      <span> {{ DisplayMetadataCtrl.collector.label }} </span>
+    </span>
+  </span>
+
+  <span ng-if="DisplayMetadataCtrl.filterElements" data="filter">
+    <span class="label" ng-class="{ 'label-primary' : true, 'label-emphasize' : true }">
```

Comment:
Do you need an `ngClass` like this `ng-class="{ 'label-primary' : true, 'label-emphasize' : true }"`?   Why not just make these regular HTML classes?
```html
<span class="label label-primary label-emphasize">
```